testing/testing_bench.ts

```diff
@@ -0,0 +1,18 @@
+import { bench, runIfMain } from "./bench.ts";
```

Comment:
Could you add a comment about the purpose of this file ? I guess this is to measure how long it takes to run the tests- as an example?